commands/commands.go

```diff
@@ -55,6 +57,13 @@ var (
 				fileLogger, _ := lumber.NewAppendLogger(filepath.ToSlash(filepath.Join(config.GlobalDir(), "nanobox.log")))
 				lumber.SetLogger(fileLogger)
 
+			} else {
+				if !strings.Contains(ccmd.CommandPath(), "server")  && util.IsPrivileged() {
```

Comment:
I will look into it. I know travis can allow you to run as admin but they dont run as admin without using 'sudo'